Ark.Tools.Nodatime.SystemTextJson/Ark.Tools.Nodatime.SystemTextJson.csproj

```diff
@@ -11,9 +11,7 @@
     <PackageReleaseNotes></PackageReleaseNotes>
 
     <Version>$(ArkToolsCoreVersion)</Version>
-    <!--<VersionSuffix>$(ArkToolsCoreVersionSuffix)</VersionSuffix>-->
```

Comment:
To add beta versions to unavailability I need beta version of this .
Is there a need to remove suffix ability again in future?

Reply:
Even with a beta-02 version listed I still have an error attempting to Install  Ark.Tools.AspNetCore 2.7.2 Beta-02
NU1102: Unable to find package Ark.Tools.Nodatime.SystemTextJson with version (>= 2.7.2)
  - Found 7 version(s) in nuget.org [ Nearest version: 2.7.2-beta02 ]

Reply:
So AspNetCore has a dependency on 2.7.2
it seems the suffix is missing somewhere else